spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -557,31 +556,31 @@ at the runtime.
 ----
 ---
 applications:
-- name: test-server
-  host: test-server
-  memory: 1G
-  disk_quota: 1G
+- name: data-flow-server
+  host: data-flow-server
+  memory: 2G
+  disk_quota: 2G
   instances: 1
-  path: spring-cloud-dataflow-server-cloudfoundry-VERSION.jar
+  path: {ABSOLUTE PATH TO SERVER UBER-JAR}
   env:
-    SPRING_APPLICATION_NAME: test-server
-    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_URL: <URL>
-    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_ORG: <ORG>
-    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_SPACE: <SPACE>
-    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_DOMAIN: <DOMAIN>
-    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_USERNAME: <USER>
-    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_PASSWORD: <PASSWORD>
+    SPRING_APPLICATION_NAME: data-flow-server
+    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_URL: https://api.local.pcfdev.io
+    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_ORG: pcfdev-org
+    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_SPACE: pcfdev-space
+    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_DOMAIN: local.pcfdev.io
+    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_USERNAME: admin
+    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_PASSWORD: admin
+    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_STREAM_SERVICES: rabbit,my-config-server
+    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_TASK_SERVICES: mysql,my-config-server
+    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_SKIP_SSL_VALIDATION: true
     MAVEN_REMOTE_REPOSITORIES_REPO1_URL: https://repo.spring.io/libs-release
-    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_STREAM_SERVICES: my-config-server #this is for all the stream applications
```

Comment:
I notice now that one of the manifest examples has comments and the other don't.  When I was looking at it, it looked actually a bit distracting to me since the line length in the docs is then very long.  Not sure it adds a ton of value.  Can go either way, but we should be consistent.